prog/minio/minio_pool_nexus.rb

```diff
@@ -17,13 +17,35 @@ def self.assemble(cluster_id, start_index)
 
       minio_pool = MinioPool.create(
         cluster_id: cluster_id,
-        start_index: start_index
+        start_index: start_index,
+        server_count: server_count,
+        drive_count: drive_count,
+        storage_size: storage_size
       ) { _1.id = ubid.to_uuid }
 
       Strand.create(prog: "Minio::MinioPoolNexus", label: "start") { _1.id = minio_pool.id }
     end
   end
 
+  def self.assemble_additional_pool(cluster_id, server_count, drive_count, storage_size)
+    unless (cluster = MinioCluster[cluster_id])
+      fail "No existing cluster"
+    end
+
+    DB.transaction do
+      minio_pool = MinioPool.create_with_id(
+        cluster_id: cluster_id,
+        start_index: cluster.target_total_server_count,
+        server_count: server_count,
+        drive_count: drive_count,
+        storage_size: storage_size
+      )
+      st = Strand.create(prog: "Minio::MinioPoolNexus", label: "start") { _1.id = minio_pool.id }
+      minio_pool.incr_additional_pool
+      st
+    end
```

Comment:
```suggestion
    DB.transaction do
      st = self.assemble(cluster_id, cluster.target_total_server_count, server_count, drive_count, storage_size)
      st.subject.incr_additional_pool
      st
    end
```

Reply:
@furkansahin Do you have any comments on this? Decided to not apply or forgot to apply?